dj_gui_api_server/DJConnector.py

```diff
@@ -96,12 +95,12 @@ def list_tables(jwt_payload: dict, schema_name: str):
                     table_name_parts[-2]) + '.' + DJConnector.snake_to_camel_case(
                         table_name_parts[-1]))
             else:
-                print(table_name + ' is of unknown table type')
+                print(table_name + ' is of unknown table type', flush=True)
 
         return tables_dict_list
 
     @staticmethod
-    def fetch_tuples(jwt_payload: dict, schema_name: str, table_name: str):
+    def fetch_tuples(jwt_payload: dict, schema_name: str, table_name: str):  # noqa: C90
```

Comment:
What is `noqa: C90`?

Reply:
It had to do with the stupid style guide complaining that the function is to complex. I am going to pull out one of the table object parsing thing as that needs to be done anyways, hopefully it will get it to stop complianing

Reply:
Abstracting the function solved the complaint